src/pspm_dcm.m

```diff
@@ -809,6 +843,7 @@
 dcm.modeltype = 'dcm';
 dcm.modality = settings.modalities.dcm;
 dcm.revision = rev;
+dcm.lasttrialfilter = error_log; % recorded the sessions that have last trial filtered

```

Comment:
I suggest to add it in the `model` structure (somewhere btw lines 623-633) because I find it natural to have the information about excluded trials in the input model for the inversion process (at the end `lasttrialfilter` will be stored in `dcm.input` as well as in `dcm.invmodel`).

Reply:
Changed it myself

Reply:
I agree with you as it should be known in the input model.